lib/google/ads/google_ads/interceptors/logging_interceptor.rb

```diff
@@ -319,11 +312,20 @@ def build_summary_message(request, call, method, is_fault)
                                 "no"
                               end
 
+            trailing_metadata = call
+              .instance_variable_get(:@wrapped)
+              .instance_variable_get(:@call)
+              .trailing_metadata
+
+            request_id = "N/A"
+            request_id = trailing_metadata["request-id"] if trailing_metadata
+
             [
               "CID: #{customer_id}",
               "Host: #{call.instance_variable_get('@wrapped').peer}",
```

Comment:
Since we're using instance_variable_get('@wrapped') twice now, maybe we could extract this out above like you do with the request_id now.

Reply:
Sure. I've extracted that out.